app/client/cypress/integration/state/links.spec.js

```diff
@@ -0,0 +1,67 @@
+describe('State Links', () => {
+  beforeEach(() => {
+    cy.visit('/state/FL');
+  });
+
+  it('Clicking the “Show more/Show less” link/button toggles between showing more/less text in the state intro paragraph', () => {
+    // verify that only "Show more" is visible
+    cy.findByText('Show less').should('not.exist');
+    cy.findByText('Show more').should('exist');
+
+    // Click "Show more" and verify that "Show less" is now visible
+    cy.findByText('Show more').click();
+    cy.findByText('Show less').should('exist');
+    cy.findByText('Show more').should('not.exist');
+
+    // Click "Show less" and verify that "Show more" is now visible
+    cy.findByText('Show less').click();
+    cy.findByText('Show less').should('not.exist');
+    cy.findByText('Show more').should('exist');
+  });
+
+  it('Clicking the “DISCLAIMER” button displays the disclaimer popup', () => {
+    // verify opening the disclaimer
+    cy.findByText('Disclaimer').click();
+    cy.findByText('The condition of a waterbody is dynamic and can change', {
+      exact: false,
+    }).should('exist');
+
+    // verify closing the disclaimer
+    cy.findByTitle('Close disclaimer').click();
+    cy.findByText('The condition of a waterbody is dynamic and can change', {
+      exact: false,
+    }).should('not.exist');
+  });
+
+  it('Clicking the “More Information for <state name>” link opens a new tab for the state', () => {
+    const linkText = 'More Information for Florida';
+
+    // cypress does not allow opening links for an external domain
+    // so we are just going to verify the anchor props
```

Comment:
That is a good point. 

The reason I had it worded that way, was because an earlier version of this test I was removing `target="_blank"` from the anchor tag and then trying to click the link. Cypress errors out when you try to open an external link in the same tab.